src/gui/PasswordWidget.cpp

```diff
@@ -233,8 +229,8 @@ void PasswordWidget::updateRepeatStatus()
 
 void PasswordWidget::autocompletePassword(const QString& password)
 {
-    if (!config()->get(Config::Security_PasswordsRepeatVisible).toBool()
-        && m_ui->passwordEdit->echoMode() == QLineEdit::Normal) {
+    // If the password is visible, always copy it into the repeat field
+    if (m_ui->passwordEdit->echoMode() == QLineEdit::Normal) {
```

Comment:
I would actually prefer not to do this. There can still be typos in the password that go unnoticed, especially if the password overflows the field width.

Reply:
So always require repeat?

Reply:
Yes.

Reply:
Doesn't bother me since this is an infrequent task. I think we should combine the warning on low entropy password with this change as well.